spec/system/school_groups/advice/scores_spec.rb

```diff
@@ -94,46 +64,44 @@
     end
   end
 
-  context 'when signed in as group admin' do
+  # Avoids problem with showing national placing. National Scoreboard only runs from
+  # 1st Sept to 31st Jul.
```

Comment:
I think the national scoreboard runs until 30th June (read it in the code a few weeks ago)